_zh-cn/overviews/scala3-book/introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 导言
+type: chapter
+description: This page begins the overview documentation of the Scala 3 language.
+num: 1
+previous-page: 
+next-page: 
```

Comment:
How is the page rendered with these empty values? Should we temporarily remove these properties?

Reply:
The original [file](https://raw.githubusercontent.com/scala/docs.scala-lang/main/_overviews/scala3-book/introduction.md):
```
---
title: Introduction
type: chapter
description: This page begins the overview documentation of the Scala 3 language.
num: 1
previous-page: 
next-page: scala-features
---
```

Reply:
We can regard each chapter/section as a node in a doubly linked list. The properties can be null but should be there.